src/plugins/controls/public/range_slider/embeddable/range_slider_embeddable.tsx

```diff
@@ -283,8 +283,8 @@ export class RangeSliderEmbeddable
     });
 
     this.dispatch.setMinMax({
-      min: `${min ?? '-Infinity'}`,
-      max: `${max ?? 'Infinity'}`,
+      min,
+      max,
```

Comment:
I think it's nicer if all of the `Infinity` and `-Infinity` defaulting happens in the **component** and not the embeddable.